dango/types/src/config.rs

```diff
@@ -26,6 +26,10 @@ pub struct AppConfig {
     /// and becomes liquidatable, liquidators can pay off the accounts debts (in return for some of
     /// its collateral) until the account's utilization rate is at this value.
     pub target_utilization_rate: Bounded<Udec128, ZeroExclusiveOneExclusive>,
+    /// The maker fee for the DEX.
+    pub maker_fee: Bounded<Udec128, ZeroInclusiveOneExclusive>,
+    /// The taker fee for the DEX.
+    pub taker_fee: Bounded<Udec128, ZeroInclusiveOneExclusive>,
```

Comment:
I'm going to rename these to `{maker,taker}_fee_rate`. The addition of word "rate" should make them less ambiguous.